hid-polostar.c

```diff
@@ -0,0 +1,212 @@
+	0x75, 0x08,         /*          Report Size (8),                        */
+	0x95, 0x01,         /*          Report Count (2),                       */
+	0x09, 0x38,         /*          Usage (Wheel),                          */
+	0x05, 0x0C,         /*          Usage Page (Consumer),                  */
+	0x0A, 0x38, 0x02,   /*          Usage (AC Pan),                         */
+	0x81, 0x06,         /*          Input (Variable, Relative),             */
+	0xC0,               /*      End Collection,                             */
+	0xC0,               /*  End Collection,                                 */
+	
+	/* Report ID 5 is used for some periphery buttons */
+	0x05, 0x0C,         /*  Usage Page (Consumer),                          */
+	0x09, 0x01,         /*  Usage (Consumer Control),                       */
+	0xA1, 0x01,         /*  Collection (Application),                       */
+	0x85, 0x05,         /*      Report ID (5),                              */
+	0x95, 0x01,         /*      Report Count (1),                           */
+	0x75, 0x08,         /*      Report Size (8),                            */
+	0x81, 0x01,         /*      Input (Constant),                           */
+	0x15, 0x00,         /*      Logical Minimum (0),                        */
+	0x25, 0x01,         /*      Logical Maximum (1),                        */
+	0x75, 0x01,         /*      Report Size (1),                            */
+	0x95, 0x12,         /*      Report Count (18),                          */
+	0x0A, 0x83, 0x01,   /*      Usage (AL Consumer Control Config),         */
+	0x0A, 0x8A, 0x01,   /*      Usage (AL Email Reader),                    */
+	0x0A, 0x92, 0x01,   /*      Usage (AL Calculator),                      */
+	0x0A, 0x94, 0x01,   /*      Usage (AL Local Machine Brwsr),             */
+	0x0A, 0x21, 0x02,   /*      Usage (AC Search),                          */
+	0x0A, 0x23, 0x02,   /*      Usage (AC Home),                            */
+	0x0A, 0x24, 0x02,   /*      Usage (AC Back),                            */
+	0x0A, 0x25, 0x02,   /*      Usage (AC Forward),                         */
+	0x0A, 0x26, 0x02,   /*      Usage (AC Stop),                            */
+	0x0A, 0x27, 0x02,   /*      Usage (AC Refresh),                         */
+	0x0A, 0x2A, 0x02,   /*      Usage (AC Bookmarks),                       */
+	0x09, 0xB5,         /*      Usage (Scan Next Track),                    */
+	0x09, 0xB6,         /*      Usage (Scan Previous Track),                */
+	0x09, 0xB7,         /*      Usage (Stop),                               */
+	0x09, 0xCD,         /*      Usage (Play Pause),                         */
+	0x09, 0xE2,         /*      Usage (Mute),                               */
+	0x09, 0xE9,         /*      Usage (Volume Inc),                         */
+	0x09, 0xEA,         /*      Usage (Volume Dec),                         */
+	0x81, 0x62,         /*      Input (Variable, No Preferred, Null State), */
```

Comment:
The whole report descriptor width is exceeding the kernel coding style's maximum of 80 columns. To reduce it the comment on this line can be wrapped.